Source/WebGPU/WebGPU/RenderPassEncoder.mm

```diff
@@ -201,10 +198,7 @@
 
 void RenderPassEncoder::setScissorRect(uint32_t x, uint32_t y, uint32_t width, uint32_t height)
 {
-    UNUSED_PARAM(x);
-    UNUSED_PARAM(y);
-    UNUSED_PARAM(width);
-    UNUSED_PARAM(height);
+    [m_renderCommandEncoder setScissorRect: { x, y, width, height } ];
```

Comment:
IIRC there are corner cases that need to be checked (like if width is negative or if x > viewport or something like that). Maybe we should add a FIXME.